eslint/babel-eslint-parser/package.json

```diff
@@ -23,7 +23,7 @@
     "./package.json": "./package.json"
   },
   "peerDependencies": {
-    "@babel/core": ">=7.11.0",
+    "@babel/core": ">=7.10.0",
```

Comment:
Why do we need to change this back to 7.10.0?

Reply:
Footgun: https://github.com/babel/babel/runs/925434089#step:4:751

Reply:
Gotcha! So will we bump this when we do the `7.11.0` release?

Reply:
I think we still support `7.10`? Maybe we should remove runtime `peerDependencies` checks, they should be addressed when installing packages.